### PR TITLE
added changed to getAllClaims to return claims for a school and mecon…

### PR DESCRIPTION
**Pull request by OlaStalberg**
…troller to catch when user not found

### CODE REVIEW

File: src/main/java/no/fint/betaling/controller/MeController.java [hunk 1]
```diff
@@ -109,11 +107,15 @@ private String getName(PersonResource person) {
...
-
-        SkoleressursResource skoleressurs = restUtil.get(SkoleressursResource.class,
-                UriComponentsBuilder.fromUriString(schoolResourceEndpoint).pathSegment("feidenavn", ePPN).build().toUriString());
+        SkoleressursResource skoleressurs;
+        try {
+            skoleressurs = restUtil.get(SkoleressursResource.class,
+                    UriComponentsBuilder.fromUriString(schoolResourceEndpoint).pathSegment("feidenavn", ePPN).build().toUriString());
+        } catch (RuntimeException ex) {
+            throw new PersonNotFoundException(ex.getMessage());
+        }
```
Comment (asgeirn) [inline src/main/java/no/fint/betaling/controller/MeController.java:118]: You should catch `HttpStatusCodeException` instead - then you can validate that the REST status code is in fact `404` and rethrow if it's not.

**Review by asgeirn — CHANGES_REQUESTED** [review]

File: src/main/java/no/fint/betaling/controller/MeController.java (hunk 1 above)
Comment (OlaStalberg) [reply src/main/java/no/fint/betaling/controller/MeController.java:118]: I will try, had an issue where the catch didnt apply when I tried earlier. Will test again and reply

Comment (OlaStalberg) [reply src/main/java/no/fint/betaling/controller/MeController.java]: @asgeirn would this be sufficient?

**Review by fsjovatsen — APPROVED** [review]

**Comment by OlaStalberg** [comment]
closes #61

**Pull request closed without merging**